features/api_enable.feature

```diff
@@ -0,0 +1,188 @@
+Feature: u.pro.services.enable
+
+    Scenario Outline: u.pro.services.enable.v1 container services
+        Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+        When I run `apt-get update` with sudo
```

Comment:
We recently landed a pr which implements `When I apt update` - we could use it here; same for similar calls